monitoring/uss_qualifier/scd/data_interfaces.py

```diff
@@ -1,12 +1,30 @@
-from typing import Optional
+from typing import Optional, List, Dict
 from monitoring.monitorlib.typing import ImplicitDict
 from monitoring.monitorlib.scd_automated_testing.scd_injection_api import InjectFlightRequest
-import datetime
+from monitoring.monitorlib import fetch
+from monitoring.uss_qualifier.rid import Severity
 
-class SCDTestIssue(ImplicitDict):
-    ''' A class to hold a message that the test executor can provide to the USS in cases when the USS provides a response that is not same as the expected result. '''
-    timestamp: Optional[str]
+class Result(object):
+    ''' A class to hold the expected output from data processing and the test interface of the USS under test '''
+    Planned = 'Planned'   
+    Rejected = 'Rejected' 
+    ConflictWithFlight = 'ConflictWithFlight'    
+    Failed = 'Failed'
+        
+class KnownIssueFields(ImplicitDict):
+    ''' A class to hold a message that the test executor can provide to the USS in cases when the USS provides a response that is not same as the expected result. '''    
+    test_code: str
+    """Code corresponding to check generating this issue"""
 
+    relevant_requirements: List[str] = []
+    """Requirements that this issue relates to"""
+
+    severity: Severity
+    """How severe the issue is"""
+
+    injection_target: str
```

Comment:
How is the simulator going to populate this field?

Reply:
My initial thinking was that this would be specified in the simulator but will remove this from the data structure